knite51/Node-Express-Sequelize-with-Postgres
# Setup Coveralls on git account

#### What does this PR do?
- Setup Coveralls on git account

#### Description of Tasks to be completed?
- Install coveralls and nyc
- Configure necessary files
- Add coveralls and nyc to test script
- Create coveralls account and link to repo
- Copy markdown url from coveralls account
- Paste markdown into READme file to display coverage build status

#### How should this be manually tested?
- Coverage build status should show on repo OR
- Run npm test on the terminal
- Coverage report should be displayed
